src/components/atoms/Error-Message.tsx

```diff
@@ -0,0 +1,10 @@
+export const ErrorMessage = () => {
+  return (
+    <div>
+      <div id="error-message" className="error-message">
+        <img src="./images/icon-error.svg" alt="" />
```

Comment:
always add a descriptive alt text to images. it is accessibility..

Reply:
> always add a descriptive alt text to images. it is accessibility..

Yes please, I will.